### PR TITLE
Adds Initial WP-CLI Cheat Sheet

### PR DESCRIPTION
**Pull request by michaelck**
# What does this PR do?

Adds a WP-CLI Cheat Sheet Goodie (wp-cli.json) with the following headings:
- Installation
- Core Commands
- Plugin Commands
- Theme Commands

-----
IA Page: https://duck.co/ia/view/wp_cli_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@michaelck Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by moollaza** [comment]
@michaelck thanks for your contribution!

You'r PR description is pretty good, but we ask that devs fill out the [pull request template](https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/pull_request_template_cheat_sheet.md) and paste it in as the PR description. Once completed, we'll be sure to review your PR and give you some feedback.

Thanks!

File: share/goodie/cheat_sheets/json/wp-cli.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
+{
+    "id": "wp_cli_cheat_sheet",
+    "name": "WP-CLI",
+    "description": "WordPress Command Line Interface",
+    "metadata": {
+        "sourceName": "WP-CLI Commands",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/wp-cli.json]: let's change this to `WP-CLI.org`

**Comment by michaelck** [comment]
**What does your Instant Answer do?**

Creates a WP-CLI Instant Answer Cheat Sheet.

**What is the data source for your Instant Answer? (Provide a link if possible)**

Data is sourced from http://wp-cli.org/

**Why did you choose this data source?**

I use WP-CLI as part of my development tool set and thought others may find it helpful. 

**Are there any other alternative (better) data sources?**

http://wp-cli.org/ is the official site / docs for the project. 

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

WordPress developers & administrators.

**Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?**

Not that I am aware.

**Are you having any problems? Do you need our help with anything?**

I was curious if Instant Answer triggers can contain hyphens. Currently visitors would have to omit the hyphen in wp-cli to trigger the cheat sheet (ex: wp cli cheat sheet).

**Where did you hear about DuckDuckHack? (For first time contributors)**

I can't recall the first time I stumbled across it, but I have intended to contribute for a while now.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

ddg-wp-cli-cheat-sheet

**Comment by michaelck** [comment]
Thanks for pointing me towards the PR Template @moollaza!

**Comment by moollaza** [comment]
@michaelck this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@michaelck congrats! It's live: https://duckduckgo.com/?q=wp-cli+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**